--- gamess.int.disabled	2016-05-31 14:13:50.209776994 +0000
+++ gamess.int	2016-05-31 14:15:41.401793068 +0000
@@ -1,7 +1,7 @@
 interfaceDescription{'gamess'}="Ab initio and semi-empirical calculations"
 interfaceDescription{'gamess_admin'}="Edit preferences specific to Gamess"
 interfaceDescription{'gamess_name'}="Gamess"
-gamessVersion="2010"
+gamessVersion="2014"
 gamessGMSPATH="/usr/local/gamess"
 gamessBinary="gamess.00.x"
 gamessDdikick="ddikick.x"
@@ -13,4 +13,4 @@
 ppnDefault="1"
 nodeTypeDefault=""
 peEnvironmentDefault=""
-resourceListDefault=""
\ No newline at end of file
+resourceListDefault=""
